sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Authoring/ConversationAuthoringClient.cs

```diff
@@ -9,6 +9,10 @@
 
 namespace Azure.AI.Language.Conversations.Authoring
 {
+    /// <remarks>
+    /// See <see href="https://docs.microsoft.com/rest/api/language/conversational-analysis-authoring"/> for more information about models you can pass to this client.
+    /// </remarks>
+    /// <seealso href="https://docs.microsoft.com/rest/api/language/conversational-analysis-authoring"/>
```

Comment:
just want to make sure this renders only as one link, and isn't actually duplicated in docs

Reply:
Nope, no duplicate. The standalone `<seealso>` would appear in a separate "See Also" section like (from some random doc):

<img width="183" alt="image" src="https://user-images.githubusercontent.com/1532486/175649644-3b4d3fa1-aed3-4b76-89ad-41d556109f58.png">